packages/block-library/src/playlist/edit.js

```diff
@@ -0,0 +1,342 @@
+			setAttributes( { [ attribute ]: newValue } );
+		};
+	}
+
+	if ( ! ids ) {
+		return (
+			<div { ...blockProps }>
+				<MediaPlaceholder
+					icon={ <BlockIcon icon={ icon } /> }
+					labels={ {
+						title: __( 'Playlist' ),
+						instructions: __(
+							'Upload an audio file or pick one from your media library.'
+						),
+					} }
+					onSelect={ onSelectTracks }
+					accept="audio/*"
+					multiple={ true }
+					allowedTypes={ ALLOWED_MEDIA_TYPES }
+					value={ attributes }
+					onError={ onUploadError }
+				/>
+			</div>
+		);
+	}
+
+	return (
+		<>
+			<BlockControls group="other">
+				<MediaReplaceFlow
+					name={ __( 'Edit' ) }
+					onSelect={ ( value ) => onSelectTracks( value ) }
+					accept="audio/*"
+					addToPlaylist={ true } // Without this, the replace flow does not fetch all the audio file data.
+					mediaIds={ ids
+						.filter( ( track ) => track.id )
+						.map( ( track ) => track.id ) }
+					multiple={ false }
+					allowedTypes={ ALLOWED_MEDIA_TYPES }
+					value={ attributes }
+					onError={ onUploadError }
+				/>
+			</BlockControls>
+			<InspectorControls>
+				<PanelBody title={ __( 'Settings' ) }>
+					<ToggleControl
+						__nextHasNoMarginBottom
+						label={ __( 'Show Tracklist' ) }
+						onChange={ toggleAttribute( 'tracklist' ) }
+						checked={ tracklist }
+					/>
+					<ToggleControl
+						__nextHasNoMarginBottom
+						label={ __( 'Show artists in Tracklist' ) }
+						onChange={ toggleAttribute( 'artists' ) }
+						checked={ artists }
+					/>
+					<ToggleControl
+						__nextHasNoMarginBottom
+						label={ __( 'Show number in Tracklist' ) }
+						onChange={ toggleAttribute( 'tracknumbers' ) }
+						checked={ tracknumbers }
+					/>
+					<ToggleControl
+						__nextHasNoMarginBottom
+						label={ __( 'Show images' ) }
+						onChange={ toggleAttribute( 'images' ) }
+						checked={ images }
+					/>
+					<SelectControl
+						label={ __( 'Order' ) }
+						value={ order }
+						options={ [
+							{ label: __( 'Descending' ), value: 'DESC' },
+							{ label: __( 'Ascending' ), value: 'ASC' },
+						] }
+						onChange={ ( value ) => onChangeOrder( value ) }
+					/>
+				</PanelBody>
+			</InspectorControls>
+			<figure { ...blockProps }>
+				<Disabled isDisabled={ ! isSelected }>
+					{ !! ids[ trackListIndex ]?.id && (
+						<div className="wp-block-playlist__current-item">
+							{ images && (
+								<img
+									src={
+										ids[ trackListIndex ]?.image.src
+											? ids[ trackListIndex ]?.image.src
+											: '/wp-includes/images/media/audio.png'
+									}
+									width="48px"
+									height="64px"
+									alt=""
+								/>
+							) }
+							<ul>
+								{ ids[ trackListIndex ]?.title && (
+									<li
+										className="wp-block-playlist__item-title"
+										dangerouslySetInnerHTML={ {
+											__html: safeHTML(
+												ids[ trackListIndex ]?.title
+											),
+										} }
+									/>
```

Comment:
`dangerouslySetInnerHTML`, `safeHTML` are needed because the Media Library and the "classic" playlist allows using some HTML here, like making the text bold.